runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -298,6 +298,7 @@ TR::CompilationInfoPerThreadRemote::processEntry(TR_MethodToBeCompiled &entry, J
    bool abortCompilation = false;
    uint64_t clientId = 0;
    TR::CompilationInfo *compInfo = getCompilationInfo();
+   auto comp = getCompilation();
```

Comment:
A comp object does not exist at this point. Most likely `comp == NULL` at this point